.github/workflows/pytest.yaml

```diff
@@ -8,18 +9,25 @@ on:
     branches:
       - main
       - release/*
+  workflow_call:
   workflow_dispatch: {}
 jobs:
   pytest:
     timeout-minutes: 30
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version:
+          - "3.9"
```

Comment:
It looks like the `install` workflow runs on Python 3.7, 3.8 and 3.9.  Why does this only run on 3.9?

Reply:
Same as above.